src/components/header/HeaderCustomButtons.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+import PriorityHighIcon from "@mui/icons-material/PriorityHigh";
+import TagFacesIcon from "@mui/icons-material/TagFaces";
+import {
+  changeEmoji,
+  changeShowDescription,
+} from "../../features/headerInfo/headerInfoSlice";
+import smile from "../smile/smiles";
+import style from "./header.module.scss";
+
+function HeaderCustomButtons() {
+  const dispatch = useDispatch();
+  const { description, emoji, showDescription } = useSelector(
+    (store) => store.headerInfo
+  );
+  const addIconJsx = () => {
+    if (!emoji) {
+      return (
+        <button
+          type="submit"
+          className={style.add_btn}
+          onClick={() => {
+            const randomEmoji = smile[Math.floor(Math.random() * 55)];
+            dispatch(changeEmoji(randomEmoji));
+          }}
+        >
+          <TagFacesIcon />
+          <div>Add Icon</div>
+        </button>
+      );
+    }
+    return null;
+  };
+
+  const addDescriptionJsx = () => {
+    let buttonTitle = "";
+    if (
+      (!!showDescription && !!description) ||
```

Comment:
use showDescription as boolean

Reply:
don't use !! in such cases

Reply:
if(showDescription) {
...
}

Reply:
if(showDescription) {
...
} else {
    if (description) {
      buttonTitle = "Show Description";
    }
    if (!description) {
      buttonTitle = "Add Description";
    }
}

write the following in one line
buttonTitle = `${description ? 'Show ' : 'Add '} Description`